### PR TITLE
Backport 'resource.kind as list' patch

### PR DESCRIPTION
**Pull request by mmazur**
aka PR #272

### CODE REVIEW

**Comment by fabianvf** [comment]
Sorry, `ResourceList`s were reimplemented in 0.9 so a straight backport of the patch won't work, you'll have to adapt it a bit. If you can get the PR working and verify that it doesn't break Ansible CI I'll happily merge though (I can also test the Ansible CI stuff, but I'm struggling to find time right now)

File: openshift/dynamic/newclient.py [hunk 1]
```diff
@@ -0,0 +1,1009 @@
+import os
```
Comment (fabianvf) [inline openshift/dynamic/newclient.py]: Did you commit this intentionally?

Comment (mmazur) [reply openshift/dynamic/newclient.py]: Heh, no, my bad. I think we have a better version of this patch somewhere, I need to talk to our guys, then I'll push it.

**Comment by mmazur** [comment]
Superseded by #277

**Pull request closed without merging**